### PR TITLE
SNOW-595309: Add long description to setup.py

### PR DESCRIPTION
**Pull request by sfc-gh-yixie**
1. Shortened the README by separating the contribution guide to file CONTRIBUTING.md
2. Add the content of README and CHANGELOG to setup.py's long description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,17 +52,20 @@ For instance,
...
+    logger = logging.getLogger(logger_name)
+    logger.setLevel(logging.DEBUG)
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.DEBUG)
+    ch.setFormatter(logging.Formatter('%(asctime)s - %(threadName)s %(filename)s:%(lineno)d - %(funcName)s() - %(levelname)s - %(message)s'))
+    logger.addHandler(ch)
 ```
 
 ## Samples
 The [Developer Guide][developer guide] and [API references][api references] have sample code.
```
Comment (sfc-gh-yixie) [inline README.md:64]: @iamontheinet Do we already have a link for the sample files? We can link here. It's no problem that it's private now.

**Review by sfc-gh-sfan — APPROVED** [review]

File: MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,11 @@
+include *.md
```
[the remark this answers is not in the document]
Comment (sfc-gh-sfan) [reply MANIFEST.in:1]: I have the same question

[the remark this answers is not in the document]
Comment (sfc-gh-yixie) [reply MANIFEST.in:1]: https://packaging.python.org/en/latest/guides/using-manifest-in/#using-manifest-in

Some file including README.md are included in the source dist by default. But CHANGELOG.md and some other files are not. So we need to have this file.

**Comment by sfc-gh-yixie** [comment]
Thanks @sfc-gh-mkeller for the comments.
This PR was merged before these issues were addressed because we needed it to test publishing to the pypi.org test site.
A follow-up PR is created #359 to address Mark's comments.

**Pull request merged**